plone/deprecated.md

```diff
@@ -0,0 +1,15 @@
+# ⚠️ DEPRECATION NOTICE ⚠️
+
+-	Plone 4.x is no longer supported.
+-	Plone 5.x is no longer supported.
+-	It is strongly recommended to migrate to the latest available Plone version. See https://plone.org/download/release-schedule for details.
+
+The official Docker images for Plone 4.x and Plone 5.x are available at https://hub.docker.com/_/plone. These images are depecrated.
+
```

Comment:
I'm sorry, but this wording is still really weird (and buries the lede) -- for context, this entire markdown file is going to be the very top of the image description _on_ https://hub.docker.com/_/plone so this whole sentence is going to read there as if it's written in the third-person.

I think the text above this already makes it clear that 4.x and 5.x are not supported, and we're already very obviously labelled a deprecation notice (and the following text is the "call to action" for where to find 6.x+), so I honestly think we should just remove this whole line:

```suggestion

```

Reply:
Thanks for the context. I agree. Committing.